samples/Meeting/src/app/views/MeetingScreen.tsx

```diff
@@ -77,6 +77,10 @@ export const MeetingScreen = (props: MeetingScreenProps): JSX.Element => {
   if (!adapter) {
     return <Spinner label={'Creating adapter'} ariaLive="assertive" labelPosition="top" />;
   }
+  const onFetchAvatarPersonaData = async (userId): Promise<AvatarPersonaData> => ({
```

Comment:
remove test code,
but would be good to have a UI test added for this feature

Reply:
@alkwa-msft think we should make a user story for some tests for the custom data model stuff?